components/ReviewCard.tsx

```diff
@@ -68,6 +81,44 @@ export const ReviewCard: React.FC<ReviewCardProps> = ({ submissionData }) => {
           </div>
           <div className="card-footer bg-white">
             {comment && <Markdown>{comment}</Markdown>}
+            <textarea
+              ref={commentTextField}
+              placeholder="Type something..."
+              style={{ width: '100%', padding: '1rem' }}
+            ></textarea>
+            <button
+              className="btn bg-success m-1 text-white"
+              onClick={async () => {
+                const textField = commentTextField.current as any
+                await accept({
+                  variables: {
+                    reviewer: Number(session.id),
+                    submissionId: submissionData.id,
+                    comment: textField.value
+                  }
+                })
+                window.location.reload()
+              }}
+            >
+              Accept
+            </button>
+
+            <button
+              className="btn bg-danger m-1 text-white"
+              onClick={async () => {
+                const textField = commentTextField.current as any
+                await reject({
+                  variables: {
+                    reviewer: Number(session.id),
+                    submissionId: submissionData.id,
+                    comment: textField.value
+                  }
+                })
+                window.location.reload()
```

Comment:
same as above, could we move this function to outside the component? Its nice to keep the components to just components, instead of being cluttered with JS logic.